src/server.ts

```diff
@@ -84,7 +85,7 @@ export const setupServer = async (agent: RestAgent, config: ServerConfig) => {
     next()
   })
 
-  app.use(errorHandler(agent.config.logger))
+  app.use(errorHandler(agent.config.logger as PinoLogger))
```

Comment:
Do we need this `as`? Can we just change the signature of `errorHandler` to take the correct credo-ts logger type

Reply:
good point, sorry missed it should have just did `git reset head main` this should be updated in the next push along with json logging